content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -71,6 +71,8 @@ For a reference to old feature gates that are removed, please refer to
 | `AnyVolumeDataSource` | `false` | Alpha | 1.18 | 1.23 |
 | `AnyVolumeDataSource` | `true` | Beta | 1.24 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
+| `ClusterTrustBundle` | `false` | Alpha | 1.28 | |
+| `ClusterTrustBundleProjection` | `false` | Alpha | 1.29 | |
```

Comment:
Please preserve the existing collation order (https://github.com/kubernetes/website/pull/41793 may help us automate sorting, but that has not yet landed).

Reply:
Done.